dev_scripts/find_images_to_rebuild.py

```diff
@@ -15,12 +16,18 @@
 os.environ["FRESHMAKER_CONFIG_FILE"] = os.path.join(sys.path[0], "config.py")
 os.environ["REQUESTS_CA_BUNDLE"] = "/etc/ssl/certs/ca-bundle.crt"
 
-from freshmaker import db, app
+from freshmaker import db, app, conf
 from freshmaker.errata import Errata, ErrataAdvisory
 from freshmaker.events import (
     ErrataAdvisoryStateChangedEvent, ManualRebuildWithAdvisoryEvent)
 from freshmaker.handlers.koji import RebuildImagesOnRPMAdvisoryChange
 
+parser = argparse.ArgumentParser()
+parser.add_argument('Errata event ID', metavar='N', type=int, nargs='*',
+                    help='Errata event ID and container images id')
```

Comment:
I think this will work. You only need to accept a single argument for the event ID.
```suggestion
parser.add_argument('errata_event_id', type=int, help='Errata event ID and container images id')
```